docs/how-to/07_running_experiments.md

```diff
@@ -14,7 +14,7 @@ all the variable(s) must be provided in the fetch treatment request body. For ea
 
 ## Running Experiments with Turing Routers
 
-When deploying a Turing router, the experiment engine can be configured to 'Turing Experiments'. For more information, check [Turing - Creating a Router](https://github.com/gojek/turing/tree/main/docs/how-to/create-a-router).
+When deploying a Turing router, the experiment engine can be configured to 'Turing Experiments'. For more information, check [Turing - Creating a Router](https://github.com/caraml-dev/turing/tree/main/docs/how-to/create-a-router).
```

Comment:
Should the URL point to docs repository now that we have a consolidated one?

Reply:
Hmmm, like the CaraML docs repository? Hmm we can do that but it wouldn't bring the user to the Turing repo's URL though, which I believe would be more helpful if the user wishes to explore what Turing is after clicking on the link?